release.py

```diff
@@ -256,11 +260,105 @@ def command_stage_docs():
 
     sys.exit(0)
 
+def command_release_announcement_email():
+    release_version_num = raw_input('What is the current release version:')
+    previos_release_version_num = raw_input('What is the previous release version:')
+    number_of_contributors = int(subprocess.check_output('git shortlog -sn --no-merges %s..%s | wc -l' % (previos_release_version_num, release_version_num) , shell=True))
+    contributors = subprocess.check_output("git shortlog -sn --no-merges %s..%s | cut -f2 | tr '\n' ',' | sed -e 's/,/, /g'" % (previos_release_version_num, release_version_num), shell=True)
+    release_announcement_data = {
+        'number_of_contributors': number_of_contributors,
+        'contributors': contributors,
+        'release_version': release_version_num
+    }
+
+    release_announcement_email = """
+        To: dev@kafka.apache.org, users@kafka.apache.org, kafka-clients@googlegroups.com
```

Comment:
This should go to `announce@apache.org` as well, that's actually the most critical one as that makes it "official".

Reply:
yes, done